lib/app/console_app.rb

```diff
@@ -62,7 +62,7 @@ def show_status
     end
 
     def register_connector
-      id = App::Config['connector_package_id']
+      id = App::Config['connector_id']
```

Comment:
you used the hash earlier. and now the string. we should use a single way

Reply:
Addressed in https://github.com/elastic/connectors-ruby/pull/194/commits/718867806e78c758c8d95ee4ae56106e8e23d4e4